client/client_update.go

```diff
@@ -71,6 +70,15 @@ func (u *CurrentUpdate) MarshalJSON() ([]byte, error) {
 	return json.Marshal(u.Provides)
 }
 
+type updateV1Body struct {
```

Comment:
Suggestion:
I'm guessing you're really just looking for a type alias?

Reply:
Essentially yes. Strictly speaking I don't even need this type at all, it's intended to make it clearer when reading, as a counterpart to `updateV2Body`. You're the one looking from the outside, so you can tell me which one you prefer. :-)

Reply:
I figured :)
And I like it, but you can just write it:
`type updateV1Body = currentUpdate` (or similar, don't remember exactly the syntax)
But this is all splitting hairs ofc